sklearn/utils/tests/test_metaestimators.py

```diff
@@ -104,3 +104,6 @@ def test_available_if_docstring():
 def test_available_if():
     assert hasattr(AvailableParameterEstimator(), "available_func")
     assert not hasattr(AvailableParameterEstimator(available=False), "available_func")
+
+    est = AvailableParameterEstimator()
```

Comment:
Please add a comment to states that this is a non regression test to make sure that decorated functions can be monkeypatched with a reference to the URL or github number of your original issue.

Reply:
Got it!